Formula/gitlint.rb

```diff
@@ -17,6 +17,8 @@ class Gitlint < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "9864ef0cc95d169a63f5ab0b4c35c2480e53be11b6787642a3e416e23ae72b3e"
   end
 
+  # Click does not support bash version older than 4.4
+  depends_on "bash"
```

Comment:
maybe we can conditionally generate bash completion depending on system bash version.